content/300-guides/050-database/800-production-troubleshooting.mdx

```diff
@@ -72,20 +72,14 @@ The following example demonstrates how to manually complete the steps of a migra
 
 ## Fixing failed migrations with <inlinecode>migrate diff</inlinecode> and <inlinecode>db execute</inlinecode>
 
-In versions 3.9.0 and later, we are offering two new commands in [Preview](/about/prisma/releases#preview) that can be used to fix failed migrations. For more information about these commands, and others, refer to our [CLI Reference documentation](/reference/api-reference/command-reference) <span class="api"></span>.
+To help with fixing a failed migration, Prisma Migrate provides the following commands for creating and executing a migration file:
```

Comment:
```suggestion
To help with fixing a failed migration, Prisma provides the following commands for creating and executing a migration file:
```
Maybe, since only one command is under `migrate` namespace